projects/lib/src/xboardengine.cpp

```diff
@@ -738,13 +750,13 @@ void XboardEngine::parseLine(const QString& line)
 	}
 	else if (command == "feature")
 	{
-		QRegExp rx("\\w+\\s*=\\s*(\"[^\"]*\"|\\d+)");
```

Comment:
I can't immediately see how this code is broken, but it probably is because detecting Xboard engines' features no longer works after this pull request.

How to test:

1. Download an engine like Fairymax that has all kinds of custom configuration options. In Ubuntu this can be installed via apt: `$ sudo apt install fairymax`
2. In Cutechess GUI go to Tools -> Settings -> Engines -> Add engine (plus sign)
3. Enter name for the engine (e.g. "Fairymax") and command ("fairymax"), and set Protocol to xboard
4. Go to the Advanced tab. With the master branch the engine options are immediately detected. With this PR there's a short wait and then nothing is detected.

So see how these options are communicated in the low level, run Fairymax from the terminal and enter the "protover 2" command. All the feature strings are printed:

```
$ fairymax
tellics say     Fairy-Max 5.0b
tellics say     by H.G. Muller
protover 2
feature myname="Fairy-Max 5.0b"
feature memory=1 exclude=1
feature setboard=0 xedit=1 ping=1 done=0
feature variants="normal,nocastle,shatranj,asean,makruk,cambodian,ai-wok,courier,knightmate,capablanca,gothic,janus,falcon,cylinder,berolina,super,seirawan,spartan,great,light-brigade,king-of-the-hill,bifurcator,team-mate,los-alamos,ciccolini,mexican,grande-acedrex,roman,almost-wildebeest,fairy"
feature option="Resign -check 0"
feature option="Resign Threshold -spin 800 200 1200"
feature option="Claim draw after -spin 50 0 200"
feature option="Ini File -file /usr/share/games/fairymax/fmax.ini"
feature option="Multi-PV Margin -spin 0 0 1000"
feature option="Variant fairy selects -combo FIDE-Clobberers /// Clobberers-FIDE /// FIDE-Nutters /// Nutters-FIDE /// Clobberers-Nutters /// Nutters-Clobberers /// FIDE-Rookies /// Rookies-FIDE /// Clobberers-Rookies /// Rookies-Clobberers /// Nutters-Rookies /// Rookies-Nutters"
feature option="Makruk rules -combo makruk /// Cambodian /// Ai-wok"
feature option="Dummy Slider Example -slider 20 0 100"
feature option="Dummy String Example -string happy birthday!"
feature option="Dummy Path Example -path ."
feature option="Automatic persistent-hash dialog -check 0"
feature option="Info -button"
feature option="Save in hash file -button"
feature option="Clear Hash -button"
feature done=1
```

Reply:
Detecting Xboard features works now.